db/migrate/20151229184253_create_token_auth_synchronizable_resources.rb

```diff
@@ -13,3 +13,13 @@ def change
     end
   end
 end
+
+if ActiveRecord::Migration.respond_to? :[]
+  class CreateTokenAuthSynchronizableResources < ActiveRecord::Migration[4.2]
```

Comment:
I did so I must be missing something because they both in the end `include` the same migration.